<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>316</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="downloadDirButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="downloadDirLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>41</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;UWP下载目录：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputDirLable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;输出文件夹：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputDirEdit">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>78</y>
      <width>451</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">rgb(255, 0, 0)</string>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="downloadDirEdit">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>39</y>
      <width>451</width>
      <height>32</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="outputDirButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>79</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="renameButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>121</y>
      <width>681</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>一键解密+整理+重命名</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="copyToOutput">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 R</family>
     </font>
    </property>
    <property name="text">
     <string>以复制文件的方式输出</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="txtFileCheckBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 R</family>
     </font>
    </property>
    <property name="text">
     <string>保存.txt文件</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>731</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <number>99</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="activityLogEdit">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 R</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="activityLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>活动日志</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="moveToOutput">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 R</family>
     </font>
    </property>
    <property name="text">
     <string>以移动文件的方式输出</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="deleteFileCheckBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 R</family>
     </font>
    </property>
    <property name="text">
     <string>重命名完成后删除源文件夹</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="spiderMode">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 R</family>
     </font>
    </property>
    <property name="text">
     <string>爬虫模式</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="localMode">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 R</family>
     </font>
    </property>
    <property name="text">
     <string>本地模式</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
